lighthouse/classes/event_properties/definitions/biosero/wells_from_destination.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+from functools import cached_property
+from lighthouse.classes.event_properties.definitions import PlateBarcode
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract
+from lighthouse.classes.services.cherrytrack import CherrytrackServiceMixin
+from lighthouse.classes.event_properties.exceptions import RetrievalError
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class WellsFromDestination(EventPropertyAbstract, CherrytrackServiceMixin):
+    def __init__(self, barcode_property: PlateBarcode):
+        self.reset()
+        self._barcode_property = barcode_property
+
+    def is_valid(self):
+        return self._barcode_property.is_valid() and (len(self._errors) == 0)
+
+    def _is_valid_destination_coordinate_not_duplicated(self, wells):
```

Comment:
I declared it as a private method because I don't want it to be used outside of this class, because I dont' want to provide any functionality with it by itself.